### DISCUSSION TITLE
Error Operation not permitted. Retrying in five minutes.

### DISCUSSION THREAD

**Discussion by langubtc** [Chia-Network/chia-blockchain#4755] [Support]
Hello  my plot server is ubuntu 12.04 and mount  windows ntfs device .But copy tmp file alert error message  
`
Could not copy "/temp/chia1/plot-k32-2021-05-13-01-44-334b63f7ce0000157356fa1774741ecafdea77a4ca320a8190f3e9dc20d16424.plot.2.tmp" to "/work/plot-k32-2021-05-13-01-44-334b63f7ce0000157356fa1774741ecafdea77a4ca320a8190f3e9dc20d16424.plot.2.tmp". Error Operation not permitted. Retrying in five minutes. 
`

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
